src/Wallabag/CoreBundle/Helper/RuleBasedTagger.php

```diff
@@ -36,6 +38,11 @@ public function tag(Entry $entry)
                 continue;
             }
 
+            $this->logger->info('Matching rule.', [
+                'rule' => $rule->getRule(),
+                'tags' => $rule->getTags(),
```

Comment:
Can't you put just string instead of the whole object?
Like the rule name and the tag list